emails/verification.spt

```diff
@@ -3,13 +3,13 @@
 [---] text/html
 % if old_email
 <p>{{ _(
-    "We've received a request to associate the email address {0} to the Liberapay account whose current address is {1}. Sound familiar?",
+    "Liberapay has received a request to change the email address for the account associated with {1} to {0}. Sound familiar?",
```

Comment:
The posessive "whose" doesnt point to a person here, which is strange. Also shorter, and clear about who "we" is. Less multi-relational too.